cmd/helm/helm.go

```diff
@@ -139,12 +139,10 @@ func newRootCmd(out io.Writer) *cobra.Command {
 		addFlagsTLS(newReleaseTestCmd(nil, out)),
 		addFlagsTLS(newResetCmd(nil, out)),
 		addFlagsTLS(newVersionCmd(nil, out)),
+
 		newCompletionCmd(out),
 		newHomeCmd(out),
 		newInitCmd(out),
-		newResetCmd(nil, out),
```

Comment:
I confirmed that all three of these are taken care of above [on lines 139-141](https://github.com/technosophos/k8s-helm/blob/1a79c28f45f64967be118d0d684008dd7e493835/cmd/helm/helm.go#L139-L141), where they're decorated with the TLS-related flags.